Sources/BraintreePayPalMessaging/BTPayPalMessagingOfferType.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+import PayPalMessages
+
+/// Preferred message offer to display
+/// - Note: This module is in beta. It's public API may change or be removed in future releases.
+public enum BTPayPalMessagingOfferType {
+
+   /// Pay Later short term installment
+   case payLaterShortTerm
+
+   /// Pay Later long term installments
+   case payLaterLongTerm
+
+   /// Pay Later deferred payment
+   case payLaterPayIn1
```

Comment:
```suggestion
   case payLaterPayInOne
```
Take it or leave it - seeing numbers in enum cases feels wrong

Reply:
Done: https://github.com/braintree/braintree_ios/pull/1153/commits/b0f635ea91decac7f88552ff291bbd4473d6e34a